src/cuttlefish_util.erl

```diff
@@ -21,7 +21,7 @@
 %% -------------------------------------------------------------------
 -module(cuttlefish_util).
 
--ifdef(TEST).
+-ifdef(EUNIT).
```

Comment:
This is inconsistent across files, some files still use TEST. Can we use TEST everywhere including the header file?